tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from blockbuster import blockbuster_ctx
+
+
+@pytest.fixture(autouse=True)
+def blockbuster(request):
+    with blockbuster_ctx("fastapi") as bb:
+        bb.functions["io.BufferedReader.read"].can_block_in(
+            "starlette/testclient.py", "receive"
```

Comment:
Starlette's  `_TestClientTransport` `receive` calls `httpx.Request.read()`. I think it should use `httpx.Request.aread()` ?